### PR TITLE
Document HA master setup

### LINKED ISSUE #948: Add Kontena HA-setup documentation
Documentation is needed how to setup high available Kontena Master.

### PR DESCRIPTION
**Pull request by jnummelin**
Added documentation how to setup master as highly available. Still bit of WIP, but all inputs already appreciated. :)

ping @kontena/core-platform 

fixes #948

### CODE REVIEW

**Comment by nevalla** [comment]
Just for curiousity, why MongoDB ports are `10481` in docker-compose.yml example?

**Comment by jnummelin** [comment]
I grabbed the mongo url from compose.io mongo replica-set and just "anonymized" the hostnames :)

File: docs/getting-started/installing/ha-master.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+---
+
+# Running Kontena master in HA setup
+
+Kontena master can be run in HA setup where multiple instances of the Kontena master form one highly available logical master.
+
+## Requirements for HA
+
+- Master nodes:
+  - Any modern Linux distribution with support for Docker and docker-compose. CoreOS and Ubuntu are more heavily tested.
```
Comment (jakolehm) [inline docs/getting-started/installing/ha-master.md]: Something is wonky here (see markdown preview)

File: docs/getting-started/installing/ha-master.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+### Setting Kontena master HA with docker-compose
+
+If you are setting up Kontena master using docker-compose you can use following configuration:
+
+```yaml
+version: '2'
+services:
+  master:
+    image: kontena/server:latest
```
Comment (jakolehm) [inline docs/getting-started/installing/ha-master.md]: Should we mention something about using "latest"? (might be a better idea to use 1.x here)

Comment (jnummelin) [inline docs/getting-started/installing/ha-master.md]: Yes, I'll change it to use 1.1 in the example and add a note about possibility (and caveats) using latest.

File: docs/getting-started/installing/ha-master.md (hunk 1 above)
Comment (jnummelin) [inline docs/getting-started/installing/ha-master.md]: There was an empty bulletpoint which made some oddness. Removing that and looks ok on `gitbook serve`

**Pull request merged**